test/MOI_wrapper.jl

```diff
@@ -1,73 +1,117 @@
-                "solve_zero_one_with_bounds_3",
-            ],
-        )
-    end
+const MOI = MathOptInterface
 
-    @testset "Continuous linear problems with $T" begin
-        MOIT.contlineartest(bridged, config)
+function runtests()
+    for name in names(@__MODULE__; all = true)
+        if startswith("$(name)", "test_")
+            @testset "$(name)" begin
+                getfield(@__MODULE__, name)()
+            end
+        end
     end
+    return
+end
 
-    @testset "ADMMIterations attribute with $T" begin
-        MOIT.linear1test(bridged, config)
-        @test MOI.get(bridged, SCS.ADMMIterations()) > 0
-    end
+test_DirectSolver() = _test_runtests(SCS.DirectSolver)
 
-    @testset "Continuous quadratic problems with $T" begin
-        MOIT.qcptest(bridged, config)
-    end
+test_IndirectSolver() = _test_runtests(SCS.IndirectSolver)
 
-    @testset "Continuous conic problems with $T" begin
-        MOIT.contconictest(bridged, config, ["rootdets", "logdets"])
-    end
+function _test_runtests(linear_solver)
+    optimizer = SCS.Optimizer()
+    MOI.set(
+        optimizer,
+        MOI.RawOptimizerAttribute("linear_solver"),
+        linear_solver,
+    )
+    MOI.set(optimizer, MOI.RawOptimizerAttribute("eps"), 1e-6)
+    MOI.set(optimizer, MOI.Silent(), true)
+    model = MOI.Bridges.full_bridge_optimizer(
+        MOI.Utilities.CachingOptimizer(
+            MOI.Utilities.UniversalFallback(MOI.Utilities.Model{Float64}()),
+            optimizer,
+        ),
+        Float64,
+    )
+    MOI.Test.runtests(
+        model,
+        MOI.Test.Config(
+            atol = 1e-4,
+            exclude = Any[
+                MOI.ConstraintBasisStatus,
+                MOI.VariableBasisStatus,
+                MOI.ConstraintName,
+                MOI.VariableName,
+                MOI.ObjectiveBound,
+            ],
+        ),
+        exclude = String[
+            # Unexpected test failures:
+            #   Failures getting ConstraintFunction
+            "test_basic_ScalarQuadraticFunction_LessThan",
+            "test_basic_VectorOfVariables_GeometricMeanCone",
+            "test_conic_SecondOrderCone_negative_initial_bound",
+            "test_model_ModelFilter_AbstractConstraintAttribute",
+            "test_model_copy_to_UnsupportedAttribute",
+            "test_modification_set_function_single_variable",
+            "test_objective_incorrect_modifications",
+            "test_solve_optimize_twice",
+            #   cone dimensions 241 not equal to num rows in A = m = 31
+            "test_conic_LogDetConeTriangle",
+            "test_conic_NormNuclearCone",
+            "test_conic_NormSpectralCone",
+            "test_conic_PositiveSemidefiniteConeSquare_3",
+            "test_conic_PositiveSemidefiniteConeSquare_VectorAffineFunction",
+            "test_conic_PositiveSemidefiniteConeSquare_VectorOfVariables",
+            "test_conic_PositiveSemidefiniteConeTriangle",
+            "test_conic_RootDetConeTriangle",
```

Comment:
@blegat These are all very weird ones. We're obviously not passing our PSD matrices correctly?

Reply:
This was a double-header. We were passing the sets incorrectly: https://github.com/jump-dev/MathOptInterface.jl/pull/1607, but we're also not scaling the off-diagonal PSD values by `sqrt(2)`.

Reply:
@blegat are there any tools for working with permuting these matrices?

We need to:
 * Convert from upper-triangular to lower-triangular
 * Scale the off-diagonal entries by `sqrt(2)`
 * Scale the corresponding solution entries by `1 / sqrt(2)`

I see there is a lot of code to do this in the old MOI wrapper, but you didn't have any in your new PR.

Reply:
I've adding the scaling. I just need to do the permutation of upper-triangular to lower-triangular